src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -149,6 +149,52 @@ cssPrefix: pf-c-code-editor
 {{/code-editor}}
 ```
 
+### With optional header content
+```hbs
+{{#> code-editor}}
+  {{#> code-editor-header}}
+    {{#> code-editor-controls}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+        <i class="fas fa-copy" aria-hidden="true"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
+        <i class="fas fa-download"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Upload code"'}}
+        <i class="fas fa-upload"></i>
+      {{/button}}
+    {{/code-editor-controls}}
+    {{#> code-header-main}}
+      {{#> l-flex}}
```

Comment:
Typically we would just put some placeholder text here so it doesn't imply the flex layout is part of or styled by the code editor. wdyt about doing that here and either creating a demo, or just showing @DaoDaoNoCode how to add it.

Reply:
Yeah, I thought about that. My reasoning was that this addition didn't necessarily warrant a demo, but it is a demo. @DaoDaoNoCode you good w/me adding the section and you using a flex layout within `__header-main`?